### DISCUSSION TITLE
[FEATUREREQUEST] Bookmarks / Links

### DISCUSSION THREAD

**Discussion by Nazgile94** [stashapp/stash#5196] [Ideas]
Hi, would be nice to have an bookmark / link feature.

get rid of P... autifill in browser :)

let stash keep manage favorites of PXXX sites instead chrome / firefox and co x)

**Comment by PhuriousGeorge** [comment]
There are already plenty of specialized local and self-hosted apps that can perform this function.

Ref:
https://github.com/awesome-selfhosted/awesome-selfhosted?tab=readme-ov-file#bookmarks-and-link-sharing

**Reply by Nazgile94** [reply]
thats not what i asked, a "all in one place" would be nice , thats the FR for.

**Reply by PhuriousGeorge** [reply]
I understand what you asked and that is precisely why I replied the way that I did.

If you want to use Stash to categorize and log your favorite links like you're proposing, create Fileless Scenes, add the hyperlink and tag it in some manner that you can find it.

Feature/software bloat is a real thing.  This causes additional time maintaining, bloated codebase and the addition of non-core features for a "convenience factor".

Why add browser hooks for 3x mainline browsers just to add a feature with very little value to the core application that already perfectly exists in any number of specialized applications?
